sdk/eventhub/event-hubs/README.md

```diff
@@ -347,12 +433,26 @@ export DEBUG=azure:event-hubs:error,azure-amqp-common:error,rhea-promise:error,r
 
 ## Next Steps
 
+### More sample code
+
 Please take a look at the [samples](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/eventhub/event-hubs/samples)
-directory for detailed examples on how to use this library to send and receive events to/from
+directory for detailed examples of how to use this library to send and receive events to/from
 [Event Hubs](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-about).
 
 ## Contributing
 
-If you'd like to contribute to this library, please read the [contributing guide](../../../CONTRIBUTING.md) to learn more about how to build and test the code.
+This project welcomes contributions and suggestions.  Most contributions require you to agree to a
+Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us
+the rights to use your contribution. For details, visit https://cla.microsoft.com.
+
+When you submit a pull request, a CLA-bot will automatically determine whether you need to provide
+a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions
+provided by the bot. You will only need to do this once across all repos using our CLA.
+
+This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
+For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
+contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
```

Comment:
@KarishmaGhiya You had some reason to replace the above with just a single sentence which links to the contributing guide right?

@willmtemple The above text is already part of the contributing guide. Thoughts on keeping the old single sentence pointing to the contributing guide?

Another mid point to consider is to the below


> This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
 For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
 contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.

> This project welcomes contributions and suggestions. If you'd like to contribute to this library, please read the [contributing guide](https://github.com/Azure/azure-sdk-for-js/blob/master/CONTRIBUTING.md) to learn more about how to build and test the code.

Reply:
Yeah I think we should use the short form on second thought. I think either is good as long as it's consistent across all the packages, and the long-form is what's in the example in Azure/azure-sdk. But, since we're using monorepo, it makes sense to have just one file with all the text and point to it from here.

Reply:
we'll leave this for now given that the other README changes include the long form, but we can reconsider at a later time